### DISCUSSION TITLE
GTK: move tab to a new window

### DISCUSSION THREAD

**Discussion by jcollie** [ghostty-org/ghostty#2625] [Feature Requests, Ideas] [answered]
Currently (with libadwaita enabled) you can move a tab to another existing window by dragging the tab header from one window to another. I propose adding a keybind action and corresponding context menu entries to move a tab to a new (previously non-existent) window where it would become the first and only tab in that window.

**Comment by mitchellh** [comment]
I'm supportive of this, and also supportive of making this a generic keybind not specific to GTK (as usual, I can do the macOS work when it comes down to it). It isn't difficult to do on macOS. 

What should we name the keybind?

**Reply by jcollie** [reply]
I was thinking `move_tab_to_new_window`. Verbose but pretty obvious.

**Comment by mitchellh** [comment, marked as answer]
#2630
